.vscode/settings.json

```diff
@@ -1,9 +1,14 @@
 {
-  "workbench.colorCustomizations": {
-    "titleBar.activeBackground": "#4459ff",
-    "titleBar.activeForeground": "#fff",
-    "titleBar.inactiveBackground": "#4459ff1a",
-    "titleBar.inactiveForeground": "#858b9a",
-    "activityBar.activeBorder": "#4459ff"
-  }
+  "tailwindCSS.experimental.configFile": {
+    "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
+    "packages/design-system-twrnc-preset/tailwind.config.js": "packages/design-system-react-native/src/**"
+  },
+  "files.associations": {
+    "*.css": "tailwindcss"
+  },
+  "editor.quickSuggestions": {
+    "strings": true
+  },
```

Comment:
Enables VS Code's autocompletion suggestions when you're typing inside string literals. [See docs](https://github.com/tailwindlabs/tailwindcss-intellisense?tab=readme-ov-file#editorquicksuggestions)